pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -530,15 +530,18 @@ def getData(self):
             if self.opts['logMode'][0]:
                 x = np.log10(x)
             if self.opts['logMode'][1]:
+                if 0 in y:
```

Comment:
This does not belong. If it's necessary to fix a bug elsewhere, then we'll need a different solution..